app/controllers/api/v2/rh_cloud/cloud_request_controller.rb

```diff
@@ -0,0 +1,82 @@
+module Api::V2::RhCloud
+  class CloudRequestController < ::Api::V2::BaseController
+    layout false
+
+    KNOWN_DIRECTIVES = {
+      'playbook-sat' => :handle_run_playbook_request,
+      'foreman_rh_cloud' => :handle_run_playbook_request,
+    }
+
+    def update
+      handler = KNOWN_DIRECTIVES[directive]
+
+      unless handler
+        render json: {
+          :message => "No valid handler is found for directive: #{directive}",
+        }, status: :bad_request
+        return
+      end
+
+      send(handler)
+
+      render json: {
+        :message => "Handled #{directive} by #{handler}",
+      }
+    end
+
+    private
+
+    def metadata
+      params['metadata']
+    end
+
+    def content
+      # the content received as base 64 of the string in double quotes
+      Base64.decode64(params['content']).tr('"', '')
+    end
+
+    def directive
+      params['directive']
+    end
+
+    def handle_run_playbook_request
+      logger.error("Playbook URL is not valid: #{content}") && return unless valid_url?(content)
+      logger.error("Reporting URL is not valid: #{metadata['return_url']}") && return unless valid_url?(metadata['return_url'])
+
+      hosts = metadata['hosts'].split(',')
+      host_ids = host_ids(hosts)
+
+      logger.warning("Some hosts were not found. Looked for: #{hosts}, found ids: #{host_ids}") unless host_ids.length == hosts.length
+
+      logger.error("sat_org_id is not present in the metadata") && return unless metadata['sat_org_id']
+      org_id = metadata['sat_org_id'].to_i
+      organization = Organization.find(org_id)
+
+      composer = nil
+      Organization.as_org(organization) do
+        composer = ::JobInvocationComposer.for_feature(
+          :rh_cloud_connector_run_playbook,
```

Comment:
probably I missed something in my setup,
I am getting the error: `"ERF42-5962 [Foreman::Exception]: No template mapped to feature Run RH Cloud playbook"`

when debugging,I see the REX feature is: 
```rb
21:08:46 rails.1   |  id: 25,
21:08:46 rails.1   |  label: "rh_cloud_connector_run_playbook",
21:08:46 rails.1   |  name: "Run RH Cloud playbook",
21:08:46 rails.1   |  description: "Run playbook genrated by Red Hat remediations app",
21:08:46 rails.1   |  provided_inputs: "playbook_url,report_url,correlation_id,report_interval",
21:08:46 rails.1   |  job_template_id: nil,
21:08:46 rails.1   |  host_action_button: false,
21:08:46 rails.1   |  notification_builder: nil,
21:08:46 rails.1   |  proxy_selector_override: nil>
```
but since `job_template` is `nil` it Immediately fails.
REX code for reference: https://github.com/theforeman/foreman_remote_execution/blob/master/app/models/job_invocation_composer.rb#L353

Reply:
the `seed` worked and now I have `job_template_id`
there was an old version of another plugin that prevented the seed.